util/init_context.go

```diff
@@ -53,7 +54,18 @@ func InitUDMContext(udmContext *context.UDMContext) {
 	udmContext.NrfUri = configuration.NrfUri
 	servingNameList := configuration.ServiceNameList
 
-	udmContext.Keys = configuration.Keys
+	udmContext.SuciProfiles = []suci.SuciProfile{
+		{
+			ProtectionScheme: "1",
```

Comment:
Is it worth mentioning why `ProtectionScheme: "1"` (standard defined value for `Profile <A>`)? Or add a reference to TS 33.501 Annex C.1?

Reply:
Agreed, I added a comment, thanks!